python3/httplib2/__init__.py

```diff
@@ -50,7 +49,25 @@
 from gettext import gettext as _
 import socket
 import ssl
-_ssl_wrap_socket = ssl.wrap_socket
+
+
+def ssl_wrap_socket(sock, key_file, cert_file, disable_validation,
+                     ca_certs, ssl_version, hostname):
+
+    if not ssl.SSLContext:
```

Comment:
sorry i gave misleading example, this will fail with AttributeError on old Pythons, hasattr is still needed

Reply:
Sorry.. I'm a spoiled Python 3 only user.. Did not know that, will fix.